### PR TITLE
fix: add email to the on prem install telemetry

### PR DESCRIPTION
**Pull request by nicufk**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by vLia — APPROVED** [review]

File: pkg/telemetry/license.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+// GetEmail returns email
+func GetEmail(license string) string {
+	if LicenseEndpoint != "" {
+		payload := EmailRequest{License: license}
+		jsonPayload, _ := json.Marshal(payload)
+		resp, err := http.Post(LicenseEndpoint, "application/json", bytes.NewBuffer(jsonPayload))
+		if err != nil {
+			return ""
```
Comment (vsukhin) [inline pkg/telemetry/license.go:29]: do we need to log errors?

File: pkg/telemetry/license.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+}
+type EmailRequest struct {
+	License string `json:"license"`
+}
+
+// GetEmail returns email
+func GetEmail(license string) string {
+	if LicenseEndpoint != "" {
+		payload := EmailRequest{License: license}
+		jsonPayload, _ := json.Marshal(payload)
```
Comment (vsukhin) [inline pkg/telemetry/license.go:26]: do we need to log errors?

File: pkg/telemetry/license.go [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+		resp, err := http.Post(LicenseEndpoint, "application/json", bytes.NewBuffer(jsonPayload))
+		if err != nil {
+			return ""
+		}
+		defer resp.Body.Close()
+
+		var emailResponse EmailResponse
+		err = json.NewDecoder(resp.Body).Decode(&emailResponse)
+		if err != nil {
```
Comment (vsukhin) [inline pkg/telemetry/license.go:35]: do we need to log errors?

**Review by vsukhin — APPROVED** [review]

File: pkg/telemetry/license.go (hunk 1 above)
Comment (nicufk) [reply pkg/telemetry/license.go:29]: it's for telemetry,and cli, we don't want for now

**Pull request merged**